mache/machines/polaris.cfg

```
@@ -0,0 +1,47 @@
+# Options related to deploying an e3sm-unified conda environment on supported
+# machines
+[e3sm_unified]
+
+# the unix group for permissions for the e3sm-unified conda environment
+group = E3SM
+
+# the path to the directory where activation scripts, the base environment, and
+# system libraries will be deployed
+base_path = /eagle/E3SM/software/e3sm-unified
+
+
+# config options related to data needed by diagnostics software such as
+# e3sm_diags and MPAS-Analysis
+[diagnostics]
+
+# The base path to the diagnostics directory
+base_path = /eagle/E3SM/diagnostics
```

I presume we don't actually want to sync over the diagnostics data here but we could if it's useful.

---

```suggestion
base_path = /lus/eagle/projects/E3SM_RRM/diagnostics
```
Still just a placeholder in case we decide to sync over diagnostics.